### PR TITLE
Ft user profile #168781754

### PR DESCRIPTION
**Pull request by alexismajya**
#### What does this PR do?
user should be able to view/edit their account profile
#### Description of Task to be completed?
- write tests for feature
- implement the feature

#### How should this be manually tested?
- clone the repository
- run npm i
- run `npm run test`
- run `npm run dev:open`
#### What are the relevant pivotal tracker stories?

[Pivotal story](https://www.pivotaltracker.com/story/show/ #168781754)

#### Screenshots (if appropriate)
Screen Shot 2019-12-14 at 16 55 52
Screen Shot 2019-12-14 at 16 57 03

### CODE REVIEW

File: src/__tests__/Components/dashboard.test.js [hunk 1]
```diff
@@ -0,0 +1,216 @@
...
+	// 	form.find('input[id="state"]').simulate('change', state);
+	// 	form.find('input[id="city"]').simulate('change', city);
+	// 	form.find('input[id="phone"]').simulate('change', phone);
+	// 	form.simulate('#btn_update', event);
+	// 	const p = wrapper.find('.repsponse');
+	// 	expect(p).toHaveLength(0);
+	// 	// const formadata = new FormData();
+	// 	// formadata.append('fullname', fullname);
+	// 	// formadata.append('address', address);
+	// });
```
Comment (lemoissonM) [inline src/__tests__/Components/dashboard.test.js]: Remove or uncomment these lines

**Review by lemoissonM — CHANGES_REQUESTED** [review]

File: src/Assets/scss/custom/userProfile.scss [hunk 2]
```diff
@@ -0,0 +1,56 @@
+
+@media screen and (max-width: 992px) {
+    .row {
+      display: block;
+    }
```
Comment (lemoissonM) [inline src/Assets/scss/custom/userProfile.scss]: be careful when modifying bootstrap classes. they may break other features

**Review by lemoissonM — CHANGES_REQUESTED** [review]

File: src/Redux/Actions/dashboard.action.js [hunk 3]
```diff
@@ -0,0 +1,42 @@
+import jwtDecode from 'jwt-decode';
+import axios from 'axios';
+import baserUrl from '../../Api/config';
+
+const token = localStorage.getItem('barefoot_token');
+let loggedUser = '';
+if (token) loggedUser = jwtDecode(token);
+
```
Comment (lemoissonM) [inline src/Redux/Actions/dashboard.action.js]: It would be better to use the existing helper function

**Review by lemoissonM — CHANGES_REQUESTED** [review]

File: src/__tests__/Components/dashboard.test.js (hunk 1 above)
Comment (alexismajya) [reply src/__tests__/Components/dashboard.test.js]: I remove it

File: src/Redux/Actions/dashboard.action.js (hunk 3 above)
Comment (alexismajya) [reply src/Redux/Actions/dashboard.action.js]: Great. done

File: src/Assets/scss/custom/userProfile.scss (hunk 2 above)
Comment (alexismajya) [reply src/Assets/scss/custom/userProfile.scss]: fixed

**Review by urbanishimwe — APPROVED** [review]

**Review by lemoissonM — APPROVED** [review]

**Pull request merged**